docs/resources/sobject.md

```diff
@@ -51,26 +51,26 @@ resource "dsm_sobject" "sobject" {
   ]
   allowed_missing_justifications = true
   rotation_policy = {
-    interval_days = 20
-    effective_at = "20231130T183000Z"
+    interval_days          = 20
+    effective_at           = "20231130T183000Z"
     deactivate_rotated_key = true
-    rotate_copied_keys = "all_external"
-}
+    rotate_copied_keys     = "all_external"
+  }
 }
 
-// copy a security object
-/*
-Copy above security object.
-When copying a security object obj_type, key_size, allowed_key_justifications_policy, allowed_key_justifications_policy,
-allowed_missing_justifications, lms or bls should not be configured.
-*/
+## copy a security object
+
+# Copy above security object.
+# When copying a security object obj_type, key_size, allowed_key_justifications_policy, allowed_key_justifications_policy, -
```

Comment:
Why the `, -` at the end of this line?

Reply:
We've also repeated `allowed_key_justifications_policy` in this line

Reply:
> Why the `, -` at the end of this line?

It's a type.

Reply:
> We've also repeated `allowed_key_justifications_policy` in this line

Removed it.